sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/src/Request/ServiceRequest.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Net;
+
+namespace Microsoft.Azure.WebPubSub.AspNetCore
+{
+    /// <summary>
+    /// Web PubSub service request.
+    /// </summary>
+    public abstract class ServiceRequest
+    {
+        /// <summary>
+        /// ConnectionContext.
+        /// </summary>
+        public ConnectionContext ConnectionContext { get; set;}
```

Comment:
This value is passed into the ctor, should it be settable?

Reply:
Change to `internal set` because rest fields in request may be deserialized from Request content and this connection context will be set  from request headers in different steps. See [HERE](https://github.com/JialinXin/azure-sdk-for-net/blob/awps-update/sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/src/Extensions/ServiceRequestExtensions.cs#L138-L141) usage.